src/components/atoms/Checkbox/Checkbox.tsx

```diff
@@ -21,6 +21,7 @@ export interface CheckboxProps
   ) => void;
 }
 
+/** @deprecated use src/components/admin/Checkbox instead */
```

Comment:
This kind of separation makes it possible to add more `@see` lines in case attendee side Checkbox is added or some other link of interest

```suggestion
/**
 * @deprecated Please use the appropriate Checkbox
 * @see src/components/admin/Checkbox
 * */
```

Reply:
modified the JSDoc: https://github.com/sparkletown/sparkle/pull/2874/commits/9f96015f2ded1d8da0274c00a2e1323adf454d70